doc/api/net.md

```diff
@@ -501,7 +507,30 @@ added: v0.5.3
 
 The amount of bytes sent.
 
-### socket.connect(options[, connectListener])
+### socket.connect()
+
+Initiate the connection on a given socket.
+
+Normally this method is not needed, the socket should be created and
+opened with [`net.createConnection()`][]. Use this only if you are
+implementing a custom Socket.
+
+Possible signatures:
+
+* [socket.connect(options[, connectListener])][`socket.connect(options)`]
+* [socket.connect(path[, connectListener])][`socket.connect(path)`]
+  for connections to UNIX domain sockets or Windows named pipes.
```

Comment:
wikipedia defines IPC as local to an OS: https://en.wikipedia.org/wiki/Inter-process_communication

> interprocess communication (IPC) refers specifically to the mechanisms an operating system provides to allow processes it manages to share data. 

Operating systems don't manage processes on remote machines, so this is same as local.

Of course, local resources can be exposed still, via various network protocols.

Note that the "remote" of RPC does *not* mean "remote host", it means "remote address space": https://en.wikipedia.org/wiki/Remote_procedure_call

> (RPC) is when a computer program causes a procedure (subroutine) to execute in another address space (commonly on another computer on a shared network)

so the remote of "RPC" is *not* the opposite of the "local" of IPC, which is confusing enough I'm starting to think "IPC socket" without the word "local" is probably best to describe these two mechanisms. Note that there are lots of other forms of IPC (sys5 semaphores, for example) that are nothing like sockets, so it needs to be "IPC socket", not just "IPC".